test/integration/externalnode_test.go

```diff
@@ -377,22 +476,41 @@ var _ = Describe(fmt.Sprintf("%s,%s: ExternalNode", focusAws, focusAzure), func(
 	})
 
 	table.DescribeTable("Testing agented and agentless configuration through CES",
-		func() {
-			expectedResult := fmt.Sprintf("virtualmachine-%s", testVm)
+		func(matchKey string) {
+			var matchValue string
+			var expectedResult []string
+
+			switch matchKey {
+			case vpcIDMatch:
+				matchValue = vpcID
+				for _, vm := range vmList {
+					expectedResult = append(expectedResult, fmt.Sprintf("virtualmachine-%s", vm))
```

Comment:
make virtualmachine const, we might use vm as prefix later

Reply:
added